app/src/main/res/values/strings.xml

```diff
@@ -26,154 +26,154 @@
-    <string name="pref_gesture_action_middle_click">Middle Click</string>
-    <string name="pref_gesture_action_right_click">Right Click</string>
-    <string name="pref_gesture_action_double_click">Double Click</string>
-    <string name="pref_gesture_action_move_pointer">Move Pointer</string>
-    <string name="pref_gesture_action_pan">Pan Local Frame</string>
-    <string name="pref_gesture_action_remote_scroll">Scroll Remote Content</string>
-    <string name="pref_gesture_action_remote_drag">Drag Remote Content</string>
-    <string name="pref_gesture_action_open_keyboard">Open Keyboard</string>
+    <string name="pref_gesture_action_left_click">Left click</string>
+    <string name="pref_gesture_action_middle_click">Middle click</string>
+    <string name="pref_gesture_action_right_click">Right click</string>
+    <string name="pref_gesture_action_double_click">Double click</string>
+    <string name="pref_gesture_action_move_pointer">Move pointer</string>
+    <string name="pref_gesture_action_pan">Pan local frame</string>
+    <string name="pref_gesture_action_remote_scroll">Scroll remote content</string>
+    <string name="pref_gesture_action_remote_drag">Drag remote content</string>
+    <string name="pref_gesture_action_open_keyboard">Open keyboard</string>
 
     <string name="pref_theme">Theme</string>
-    <string name="pref_theme_option_system">Follow System Theme</string>
+    <string name="pref_theme_option_system">System</string>
     <string name="pref_theme_option_dark">Dark</string>
     <string name="pref_theme_option_light">Light</string>
-    <string name="pref_prefer_advanced_editor">Prefer Advanced Editor</string>
-    <string name="pref_prefer_advanced_editor_summary">Enable this to directly open Advanced mode to add/edit Servers</string>
-    <string name="pref_sort_server_list">Sort Server List</string>
-    <string name="pref_sort_server_list_summary">Servers are sorted by their name</string>
+    <string name="pref_prefer_advanced_editor">Prefer advanced editor</string>
+    <string name="pref_prefer_advanced_editor_summary">Opens advanced mode when adding/editing servers</string>
+    <string name="pref_sort_server_list">Sorting of servers</string>
+    <string name="pref_sort_server_list_summary">By name</string>
     <string name="pref_gesture">Gestures</string>
-    <string name="pref_single_tap">Single Tap</string>
-    <string name="pref_two_finger_tap">2 Finger Tap</string>
+    <string name="pref_single_tap">Single tap</string>
+    <string name="pref_two_finger_tap">Two-finger Tap</string>
     <string name="pref_double_tap">Double Tap</string>
-    <string name="pref_long_press">Long Press</string>
-    <string name="pref_swipe1">1 Finger Swipe</string>
-    <string name="pref_swipe2">2 Finger Swipe</string>
-    <string name="pref_drag">Drag (Long Press then Swipe)</string>
+    <string name="pref_long_press">Long press</string>
+    <string name="pref_swipe1">1 Finger swipe</string>
```

Comment:
```suggestion
    <string name="pref_swipe1">One-finger swipe</string>
```

Reply:
I originally used numerals here to more clearly distinguish (at first glance) between gestures. [Material design](https://material.io/design/communication/writing.html#principles) also suggests the same. 
Maybe we can use `1-finger`?

Reply:
Does anything actually use "1-finger" though?
You have x messages is easy, because it enumerates.